src/QsCompiler/Tests.Compiler/TestCases/QirTests/TestCaching2.ll

```diff
@@ -29,7 +29,7 @@ condContinue__2:                                  ; preds = %condFalse__2, %cond
   %7 = insertvalue %Range %6, i64 %2, 0
   %8 = insertvalue %Range %7, i64 2, 1
   %9 = insertvalue %Range %8, i64 %5, 2
-  %10 = call %Array* @__quantum__rt__array_slice_1d(%Array* %vals, %Range %9)
+  %10 = call %Array* @__quantum__rt__array_slice_1d(%Array* %vals, %Range %9, i1 true)
```

Comment:
```suggestion
  %10 = call %Array* @__quantum__rt__array_slice_1d(%Array* %vals, %Range %9, i1 zeroext true)
```
(more [here](https://llvm.org/docs/LangRef.html#parameter-attributes) about the `zeroext`)

Reply:
this is autogenerated, and the change to do that across the board will be done separately (and not included in this release)